plugins/frontend/backstage-plugin-github-insights/src/hooks/useProjectName.ts

```diff
@@ -19,6 +19,5 @@ import { Entity } from '@backstage/catalog-model';
 
 export const GITHUB_INSIGHTS_ANNOTATION = 'github.com/project-slug';
 
-export const useProjectName = (entity: Entity) => {
-  return entity?.metadata.annotations?.[GITHUB_INSIGHTS_ANNOTATION] ?? '';
-};
+export const isGithubInsightsAvailable = (entity: Entity) =>
```

Comment:
I know it's a hassle but I think isGithubInsightsAvailable should live elsewhere. src/utils.ts ? or src/isGithubInsightsAvailable.ts ?

Reply:
Sure thing!